magento/magento2
# feat: disable the setup route by default for nginx

### Description (*)
Add access control to the `/setup/` route within the default nginx configuration. 
With a default of deny all.

### Fixed Issues (if relevant)
1. Closes: #39227 

### Manual testing scenarios (*)
Test on BOTH a new uninstantiated instance, as well as a preinstalled instance. 

1. Navigate to `/setup/` expect to see a 403/401 error
2. Add your IP Address to the nginx config & reload
3. Navigate to `/setup/` expect to see the setup page.

### Questions or comments
Happy for this to be closed another way (eg removal of the route all together). 

### Contribution checklist (*)
